distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
```

Comment:
I would say this is used when performing any federated request, not just queries. Queries, resource, retrieval, etc. will all federate identity.

Reply:
Turn the dash in "queries - the" into a period. "queries. The"